.github/workflows/tag.yml

```diff
@@ -27,7 +27,7 @@ on:
         description: 'Line containing the version string'
         type: string
         required: false
-        default: "^\\s*gradle.ext.releaseVersion\\s*=\\s*'.*'"
+        default: "^\\s*gradle.ext.version\\s*=\\s*'.*'"
```

Comment:
out of curisosity, why change the default here?

Reply:
this is a reusable workflow file, and ideally can be shared by multiple repos
(reusable repos will be moved to DSP/github-actions). Current setup has a different version variable in each repo, changed this default to 'version' as a potential version variable across repos once this is used in others